go/externals/proof_service_parameterized.go

```diff
@@ -45,10 +45,8 @@ type ParamProofServiceType struct {
 
 func NewParamProofServiceType(conf keybase1.ParamProofServiceConfig) ParamProofServiceType {
 	return ParamProofServiceType{
-		conf: conf,
-		// TODO the loader will have to validate the config from the server and
-		// disable invalid ones without blowing up. CORE-8655
-		usernameRe: regexp.MustCompile(conf.UsernameRe),
+		conf:       conf,
+		usernameRe: regexp.MustCompile(conf.Username.Re),
```

Comment:
gotta not crash

Reply:
we validate here and skip anything that doesn't compile: https://github.com/keybase/client/pull/13791/files#diff-b29a26bcb8c6f6b482c2093be8890502R149 

i wanted to avoid making an intermediate type to pass the compiled version down and have the `MustCompile` as a final sanity check

Reply:
I'm kind of a fan of untrusted and validated types. Ok though put comment here saying where it was checked

Reply:
if there are any other validations steps we can make a new type, i'll add it in the comment